### DISCUSSION TITLE
ESP32S3 and ILI9488 SPI

### DISCUSSION THREAD

**Discussion by smkr9500** [Bodmer/TFT_eSPI#2955] [Q&A - General]
Hi

I can't find a file for ESP32S3 and ILI9488 SPI! But the file "Setup70b_ESP32_S3_ILI9341.h" can be used too with ILI9488!

I did this:
`
#define USER_SETUP_ID 70
// See SetupX_Template.h for all options available
**//#define ILI9341_DRIVER
#define ILI9488_DRIVER**

                    // Typical board default pins - change to match your board
#define TFT_CS   10 //     10 or 34 (FSPI CS0) 
#define TFT_MOSI 11 //     11 or 35 (FSPI D)
#define TFT_SCLK 12 //     12 or 36 (FSPI CLK)
#define TFT_MISO 13 //     13 or 37 (FSPI Q)

// Use pins in range 0-31
#define TFT_DC    7
#define TFT_RST   6`

That way I just have to comment/uncomment the driver line depending on project.

Is that wrong?

**Comment by Bodmer** [comment]
If it works, then that is a good approach.

**Comment by shibbidydibbidy** [comment]
I have the ESP32_S3 and a ILI9488 and this is not working for me. Which ESP32_S3 were you using?
